CRUDRecipeEF.PL/Menus/IngredientMenu.cs

```diff
@@ -0,0 +1,184 @@
+﻿using CRUDRecipeEF.BL.DL.DTOs;
+using CRUDRecipeEF.BL.DL.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.PL.Menus
+{
+    public class IngredientMenu : IIngredientMenu
+    {
+        private readonly IIngredientService _ingredientService;
+
+        private enum IngredientMenuOption { InValid = 0, NewIngredient = 1, LookUpIngredient = 2, ShowIngredient = 3, DeleteIngredient = 4, GoBack = 5 };
+
+        public IngredientMenu(IIngredientService ingredientService)
+        {
+            _ingredientService = ingredientService;
+        }
+
+        public async Task Show()
+        {
+            ConsoleHelper.DefaultColor = ConsoleColor.Blue;
+            ConsoleHelper.ColorWriteLine(ConsoleColor.Yellow, "Ingredient Menu");
+            Console.WriteLine();
```

Comment:
Use /n instead of empty writelines

Reply:
I'm a huge fan of empty Console.WriteLines() since allows the source code to look more similar to the ouput, however I can change this to \n or Environmen.NewLines if you prefer.

Reply:
It just adds unneccesary lines of codes but if you prefer it keep it. Considering its your main menu and it doesnt affect the app nor a big impact on readability

Reply:
Ok, I do want to conform to the style you would prefer though. If you would like this changed at all let me know, otherwise it is ready to merge.